code/game/objects/items/toys.dm

```diff
@@ -1556,3 +1556,21 @@
 	throwforce = 0
 	breakouttime = 0
 	ignoresClumsy = TRUE
+/obj/item/toy/plushie/crow
+	name = "Crow Plushie"
+	desc = "When your mad at the world, full of stress, just squeeze this crow for reassuring caws!"
+	icon = 'icons/obj/toy.dmi'
+	icon_state = "corvid_plushie"
+	attack_verb = list("pecked", "clawed", "wing slapped")
+	var/supercaw = 'sound/creatures/caw.ogg'
+	resistance_flags = FLAMMABLE
```

Comment:
Flag already assigned on the parent.

Reply:
Your like a miracle worker, you know all of the parents ha ha!

Reply:
Anyhow I have committed all of your suggestions, I still need to test and redo some pixel art. As per making an actual plushie.